src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -305,7 +305,7 @@ def test_WINNF_FT_S_WDB_2(self, config_filename):
     cpi_database.setFilesToServe(files)
 
     # Inject the CPI database URL into the SAS UUT.
-    self._sas_admin.InjectDatabaseUrl(config['cpiDatabaseConfig']['indexUrl'])
+    self._sas_admin.InjectDatabaseUrl({'type': 'CPI', 'url': config['cpiDatabaseConfig']['indexUrl']})
```

Comment:
I think we need one more fix here so that we are sure to inject the full URL, not just the relative path, and then you will have addressed all of Javi's bugs :)

Reply:
fixed.